generate-and-run-shaders/src/main/java/com/graphicsfuzz/generator/ShaderProducer.java

```diff
@@ -84,7 +84,7 @@ public void run() {
             ns.get("aggressively_complicate_control_flow"),
             ns.get("replace_float_literals"),
             donorsDir,
-            ns.get("generate_uniform_bindings"),
+            ns.get("vulkan"),
```

Comment:
Comment - no action needed: despite my request for the boolean parameter to be named `isVulkan` I think that `vulkan` is still the right name for the command line argument.

Reply:
check